doc/rippled-example.cfg

```diff
@@ -609,6 +609,15 @@
 #
 #
 #
+# [validator_key_revocation]
+#
+#   If a validator's keys have been compromised, a revocation must be generated
```

Comment:
Okay, that makes sense.  It feels like we should recommend that the revoked public key be removed from the validators.txt when the new public key is added.  The `[validator_key_revocation]` value is a good way to get the word out fast.  But if we leave revoked keys in the validators.txt file that makes a maintenance problem.